docs/topics/js/js-modules.md

```diff
@@ -8,14 +8,14 @@ following configurations for JavaScript modules:
 - [Asynchronous Module Definitions (AMD)](https://github.com/amdjs/amdjs-api/wiki/AMD), which is in particular
    used by the [RequireJS](https://requirejs.org/) library.
 - [CommonJS](http://wiki.commonjs.org/wiki/Modules/1.1), widely used by Node.js/npm
-   (`require` function and `module.exports` object)
+   (`require` function and `module.exports` object).
 - Plain. Don't compile for any module system. You can access a module by its name in the global scope.
 
 ## Browser targets
  
 If you're targeting the browser and want to use a different module system than UMD, you can specify the desired module
```

Comment:
```suggestion
If you're targeting the browser and want to use a module system other than UMD, you can specify the desired module
```

Reply:
It might be even better to restructure this a little to something like:
"If you intend to run your code in a web browser environment, and want to use a module system..."